src/gc/impl/conservative/gc.d

```diff
@@ -2078,6 +2078,135 @@ struct Gcx
     ToScanStack!(ScanRange) toscanPrecise;
 
     /**
+     * Finds the offset between an index and the next set bit, up to our max.
+     *
+     * Returns: The offset between index and the next bit,
+     *          or -1 if no bits are set between index and max
+     */
+    int bitScanForwardOffset(size_t* bitArray, size_t index, size_t max) nothrow
```

Comment:
>The compiler can replace the entire call to bsf with an assembly instruction
That's good to know! Where does it do that? I didn't see that happening in core.bitop.